scripts/supervisord/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# This is an "initContainer" image using the base "source-to-image" OpenShift template
+# in order to apprpriately inject the supervisord binary into the application container.
+#
+# We use the base s2i image in order to use binaries such as "fix-permissions" to
+# add approriate file/folder permissions to nodejs / ruby / python applications.
+# 
+
+FROM centos/s2i-base-centos7
+
+ENV SUPERVISORD_DIR /opt/supervisord
+
+RUN mkdir -p ${SUPERVISORD_DIR}/conf ${SUPERVISORD_DIR}/bin
+
+ADD supervisor.conf ${SUPERVISORD_DIR}/conf/
```

Comment:
Why have you removed the possibility to customize the commands exposed by the supervisord as we did [here](https://github.com/snowdrop/k8s-supervisor/blob/master/supervisord/conf/supervisord.tmpl#L1-L6) and populated using env var and by example this string pf CMDS `run-java:/usr/local/s2i/run;compile-java:/usr/local/s2i/assemble` ?

Reply:
I would like from this PR to just migrate what we are doing currently to initContianer approach with Go SupervisorD. And nothing else. So we can keep this PR small or smallish. 
We can work on improvements like this afterward.